packages/ui/src/components/Catalog/CatalogFilter.tsx

```diff
@@ -51,6 +65,25 @@ export const CatalogFilter: FunctionComponent<CatalogFilterProps> = (props) => {
             </ToggleGroup>
           </FormGroup>
         </GridItem>
+        <GridItem md={2} lg={1}>
+          <FormGroup label="Layout" fieldId="layout">
+            <ToggleGroup aria-label="Change layout">
+              {props.layouts.map((key) => (
+                <ToggleGroupItem
+                  icon={getCatalogLayoutIcon(key)}
```

Comment:
I think that in this case it won't matter much be it would be better to avoid calling the `getCatalogLayoutIcon` function. An alternative would be to transform the `getCatalogLayoutIcon` function into a component and use the `key` as a `prop` to render the right icon.

The reasoning is that the only way to know what `getCatalogLayoutIcon` would return, is to execute the function every time.

I'm thinking in something like the following

```jsx

interface CatalogLayoutIconProps {
  layout: CatalogLayout;
}

const CatalogLayoutIcon: FunctionComponent<CatalogLayoutIconProps> = (props): ReactNode => 
  return props.layout === CatalogLayout.Gallery ? <GripHorizontalIcon /> : <BarsIcon />;
};
```

Or maybe keeping the switch and providing a default value just in case the Layout toggle grows in the future, but better to wait until that point instead.

The the usage would be:

```jsx
                <ToggleGroupItem
                  icon={<CatalogLayoutIcon key={key} layout={key} />}
```